src/contracts/user-deployer-whitelist.ts

```diff
@@ -48,13 +62,23 @@ export class UserDeployerWhitelist extends Contract {
    * @param deployer Deployer account address.
    * @param tierId
    */
-  addDeployerAsync(deployer: Address, tierId?: TierID): Promise<void> {
+  addDeployerAsync(deployer: Address, tierId: TierID): Promise<void> {
```

Comment:
Why did this change?

Reply:
I was just thinking, to make it necessary to specify TierID at the time of AddUserDeployer.
But it is optional, as It was confirmed that it will be optional argument and will take default value if not specified, same condition is replicated here.